pom.xml

```diff
@@ -315,7 +315,15 @@
           <version>${maven-surefire-plugin.version}</version>
           <configuration>
             <redirectTestOutputToFile>true</redirectTestOutputToFile>
+            <trimStackTrace>false</trimStackTrace>
           </configuration>
+          <dependencies>
+            <dependency>
+              <groupId>org.ow2.asm</groupId>
+              <artifactId>asm</artifactId>
+              <version>7.0-beta</version>
```

Comment:
I'm not familiar with this dependency.  What does this provide?

Reply:
Added comment